jsonnet/kube-prometheus/components/grafana.libsonnet

```diff
@@ -22,6 +22,10 @@ local defaults = {
   },
   prometheusName: error 'must provide prometheus name',
   dashboards: {},
+  // TODO(paulfantom): expose those to have a stable API. After kubernetes-grafana refactor those could probably be removed.
```

Comment:
Do we have issue for this? Would be great to have this as a contribution from outside maybe, wdyt?

Reply:
We do not have it. I'll create one after this is merged.